operations/helm/charts/mimir-distributed/capped-small.yaml

```diff
@@ -1,65 +1,72 @@
-# These values configure the Grafana Mimir or Grafana Enterprise Metrics cluster to
-# handle production ingestion of ~1M series using the blocks storage engine scraped
-# with a 15s interval.
+# These values configure the Grafana Mimir or Grafana Enterprise Metrics cluster
+# for a more production-ready setup. The setup targets 70% CPU and memory utilization
+# so that the cluster has room to grow. The resource requests and limits reflect 100% utilization.
+# This is useful when you want to strictly control costs when purchasing GEM through a marketplace subscription for example.
+# limits==requests also ensure a "Guaranteed" QoS: https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/.
+# Note that setting CPU limits has some known issues:
+# https://engineering.indeedblog.com/blog/2019/12/unthrottled-fixing-cpu-limits-in-the-cloud/.
+#
+# These values are suitable for ingestion of ~1M series and scrape interval of 15s.
+# This implies ingestion rate of around 66000 samples per second.
+#
 # Query requirements can vary dramatically depending on query rate and query
-# ranges. The values here satisfy a "usual" query load as seen from our
-# production clusters at this scale.
-# It is important to ensure that you run no more than one ingester replica
-# per node so that a single node failure does not cause data loss. Zone aware
-# replication can be configured to ensure data replication spans availability
-# zones. Refer to [Zone Aware Replication](https://grafana.com/docs/mimir/v2.0.x/operators-guide/configuring/configuring-zone-aware-replication/)
-# for more information.
-# Minio is no longer enabled and you are encouraged to use your cloud providers
-# object storage service for production deployments.
-# Ingesters are configured with a replication factor of 3 to ensure that a single ingester failure does not
-# result in data loss.
+# ranges. The values here satisfy a "usual" query load of around 5 queries per second
+# as seen from our production clusters at this scale.
+#
+# The values in this file also add podAntiAffinity rules for ingesters and store-gateways.
+# The rules ensure that the replicas of the same component are not scheduled on the same
+# Kubernetes Node. Coming soon: guides for production Mimir and GEM setups, and for
+# configuring zone-aware replication in Helm.
+#
+# MinIO is no longer enabled, and you are encouraged to use your cloud providers
+# object storage service such as S3 or GCS.
 
 alertmanager:
   persistentVolume:
     enabled: true
-  replicas: 3
+  replicas: 1
```

Comment:
I think we should have 2. I'm worried that if the single alertmanager is shut down, but k8s cannot schedule the new one, the user is stuck without an alertmanager.